Framework/PythonInterface/plugins/algorithms/FitIncidentSpectrum.py

```diff
@@ -30,6 +30,9 @@ def summary(self):
                'Outputs a workspace containing the functionalized fit and its first ' \
                'derivative.'
 
+    def seeAlso(self):
```

Comment:
I don't think there is any need to define this function when it returns an empty list.